x/leverage/keeper/keeper.go

```diff
@@ -375,9 +371,6 @@ func (k Keeper) SetCollateralSetting(ctx sdk.Context, borrowerAddr sdk.AccAddres
 
 // GetCollateralSetting checks if a uToken denom is enabled for use as collateral by a single borrower.
 func (k Keeper) GetCollateralSetting(ctx sdk.Context, borrowerAddr sdk.AccAddress, denom string) bool {
-	if !k.IsAcceptedUToken(ctx, denom) {
-		return false
-	}
```

Comment:
Why was this removed?

Reply:
Redundant check mostly - a collateral setting would only be true if an asset was acceptable when it was enabled.

In the cases where the asset is no longer accepted (specifically lending disabled or blacklisted now) retuning true if a 0x01 was stored in state doesn't seem harmful.